packages/api/src/services/location.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import { GeoLocation } from '@sf/core/location';
+
+export interface LocationService {
+  getGeoLocation(address: string): Promise<GeoLocation>;
+}
+
+export const locationService: LocationService = new (class implements LocationService {
+  async getGeoLocation(address: string): Promise<GeoLocation> {
+    const url = `https://api.mapbox.com/geocoding/v5/mapbox.places/${
+      encodeURIComponent(address)
+    }.json?limit=1&access_token=${process.env.MAP_BOX_ACCESS_TOKEN}`;
+    const { data: { features } } = await axios.get(url);
+    // TODO: Investigate error response types (including RATE_LIMIT) of Mapbox API and handle them.
```

Comment:
How do you manage `TODO` in a real-world project?

Reply:
I will create a new ticket or task to handle this `TODO` and allocate some time to work on that.
It depends on the priority and if it's critical or not.